Configuration/Geometry/python/dict2021Geometry.py

```diff
@@ -1632,13 +1665,13 @@
     ("O5","T5","C1","M1","F1","P7") : "2021FlatMinus05Percent",
     ("O5","T6","C1","M1","F1","P7") : "2021FlatMinus10Percent",
     ("O5","T7","C1","M1","F1","P7") : "2021FlatPlus05Percent",
-    ("O5","T8","C1","M1","F1","P7") : "2021FlatPlus10Percent",
-    ("O6","T3","C1","M2","F1","P7") : "2023",
-    ("O4","T4","C1","M2","F1","P7") : "2023ZeroMaterial",
-    ("O5","T5","C1","M2","F1","P7") : "2023FlatMinus05Percent",
-    ("O5","T6","C1","M2","F1","P7") : "2023FlatMinus10Percent",
-    ("O5","T7","C1","M2","F1","P7") : "2023FlatPlus05Percent",
-    ("O5","T8","C1","M2","F1","P7") : "2023FlatPlus10Percent",
+    ("O5","T8","C1","M1","F2","P7") : "2021FlatPlus10Percent",
```

Comment:
Is this line expected to be changed? From PR description and update of README, it seems to effect 2023 only.

Reply:
This was indeed the right correction. All 2021 scenarios should use M1 rather than M2. That was a bug fix for a variant 2021 scenario. The main changes were for 2023 scenarios.

Reply:
Then, please fix readme file. Because it is currently,
`2021FlatPlus10Percent = T8+C1+M1+P7+O5+F1`

And better short clarification after, like 
`2021FlatPlus10Percent = T8+C1+M1+P7+O5+F2 (short explanation here)`

Thanks.

Reply:
I have changed the one for 2021FlatPlu10Percent